addons/mail/static/src/messaging/component/messaging_menu/messaging_menu_tests.js

```diff
@@ -21,11 +21,12 @@ QUnit.module('MessagingMenu', {
             if (this.widget) {
                 this.widget.destroy();
             }
-            let { discussWidget, widget } = await utilsStart(Object.assign({}, params, {
+            let { discussWidget, env, widget } = await utilsStart(Object.assign({}, params, {
                 data: this.data,
                 hasMessagingMenu: true,
             }));
             this.discussWidget = discussWidget;
+            this.env = env;
```

Comment:
I think I added it with my changes yesterday, it was used and then reverted. Diff on env can probably be removed.

Reply:
`this.env` is used in tests, so this diff is important